tests/kernels/moe/test_pplx_cutlass_moe.py

```diff
@@ -0,0 +1,362 @@
+# SPDX-License-Identifier: Apache-2.0
+import dataclasses
+import traceback
+from typing import Callable
+
+import pytest
+import torch
+
+from vllm import _custom_ops as ops
+from vllm.config import VllmConfig, set_current_vllm_config
+from vllm.model_executor.layers.activation import SiluAndMul
+from vllm.model_executor.layers.fused_moe.cutlass_moe import CutlassExpertsFp8
+from vllm.model_executor.layers.fused_moe.fused_moe import fused_topk
+from vllm.model_executor.layers.fused_moe.modular_kernel import (
+    FusedMoEModularKernel)
+from vllm.model_executor.layers.fused_moe.pplx_prepare_finalize import (
+    PplxPrepareAndFinalize)
+from vllm.platforms import current_platform
+
+try:
+    from pplx_kernels import AllToAll  # or AllToAllInternode?
```

Comment:
IIUC, `AllToAll` dispatches to `AllToAllInternode` under the hood, so we shouldn't need to interact with it directly